src/recipe/Baked.Recipe.Service.Application/Core/Dotnet/DotnetCoreExtensions.cs

```diff
@@ -1,10 +1,19 @@
 ﻿using Baked.Core;
 using Baked.Core.Dotnet;
+using System.Reflection;
 
 namespace Baked;
 
 public static class DotnetCoreExtensions
 {
-    public static DotnetCoreFeature Dotnet(this CoreConfigurator _) =>
-        new();
+    public static DotnetCoreFeature Dotnet(this CoreConfigurator _,
+        Assembly? entryAssembly = default,
+        Func<Assembly, string?>? baseNamespace = default
+    )
+    {
+        entryAssembly ??= Assembly.GetEntryAssembly() ?? Assembly.GetCallingAssembly();
```

Comment:
calling assembly usage is extremely dangerous since it only returns the assembly that calls this method which is unpredictable and thus should not be used as a default in this library